aepsych/generators/random_generator.py

```diff
@@ -59,3 +60,42 @@ def from_config(cls, config: Config):
         ub = config.gettensor(classname, "ub")
         dim = config.getint(classname, "dim", fallback=None)
         return cls(lb=lb, ub=ub, dim=dim)
+
+class AxRandomGenerator(AEPsychGenerationStep):
+    classname = "RandomGenerator"
+    model = Models.UNIFORM
+    def __init__(
+        self,
+        lb: Union[np.ndarray, torch.Tensor],
+        ub: Union[np.ndarray, torch.Tensor],
+        dim: Optional[int] = None,
+    ):
+        """Iniatialize RandomGenerator.
+        Args:
+            lb (Union[np.ndarray, torch.Tensor]): Lower bounds of each parameter.
+            ub (Union[np.ndarray, torch.Tensor]): Upper bounds of each parameter.
+            dim (int, optional): Dimensionality of the parameter space. If None, it is inferred from lb and ub.
+        """
+
+        self.lb, self.ub, self.dim = _process_bounds(lb, ub, dim)
+        self.bounds_ = torch.stack([self.lb, self.ub])
+
+    @classmethod
+    def get_config_options(cls, config: Config, name: str) -> Dict:
+        
+        # config_opts = super().get_config_options(config, name)
+        # lb = 
+        opts = {
+            "model": cls.model,
+        }
```

Comment:
You should also have key named `"model_kwargs"` in this dict. The value should be the arguments that `Models.UNIFORM` takes. I think at a minimum it takes a seed, but you should double check the Ax documentation. Look at what `AxSobolGenerator` does.